cooker/lib/pages.js

```diff
@@ -17,6 +18,21 @@ const { exists } = require("fs");
 // consts
 const env = getClientEnvironment();
 
+const objectFilter = function (obj, keys) {
```

Comment:
Would you mind just adding a quick comment here on what this function does? It's easy to understand by looking at its usage at the bottom of the file, but it would be nice just to have a one-liner above this declaration too.

Reply:
Try that now. Let me know if that's clear enough.

Reply:
Thank you!